LINK/etc/default/evm

```diff
@@ -3,11 +3,11 @@
 # Description: Sets the environment for scripts and console users
 #
 
+# for all properties files, read the lines and export all the lines that are not blank, or a # / ; comment line
```

Comment:
is this comment correct?  I don't see `/` in the new regex. I can't tell if the comment or code is wrong or if I'm not understanding.

Reply:
@jrafanie it is for `#` _**OR**_ `;`.  The `/` is being used as a separator like it would be in "and/or".

Reply:
Possibly not using a `/` would make that more clear, and I will take credit for (poorly) suggesting `#/;` in the first place:

https://github.com/ManageIQ/manageiq-appliance/pull/329#discussion_r704794724

Reply:
haha, yeah, I didn't think it supported `//` as a comment but figured I'd ask